tests/Aql/BlockFetcherHelper.cpp

```diff
@@ -213,11 +213,12 @@ std::pair<ExecutionState, AqlItemMatrix const*> AllRowsFetcherHelper::fetchAllRo
 // - SECTION CONSTFETCHER              -
 // -----------------------------------------
 
-ConstFetcherHelper::ConstFetcherHelper(std::shared_ptr<VPackBuffer<uint8_t>> vPackBuffer)
+ConstFetcherHelper::ConstFetcherHelper(AqlItemBlockManager& itemBlockManager,
+                                       std::shared_ptr<VPackBuffer<uint8_t>> vPackBuffer)
     : ConstFetcher(),
       _vPackBuffer(std::move(vPackBuffer)),
-      _resourceMonitor(),
-      _itemBlockManager(&_resourceMonitor),
```

Comment:
is it really necessary to store references to both, instead of just the itemBlockManager ?

Reply:
Actually, we don't need either.